project/src/test/java/com/google/sps/cloudstoragetests/GcsFilePathTest.java

```diff
@@ -54,16 +54,16 @@ public void setUp() {
     }
     
     @Test
-    public void EmptyMapForGcsFilePath() {
-        String actual = getGcsFilePath(NO_FILES);
+    public void emptyMapForGetGcsFilePath() {
```

Comment:
I think these all look good, but can you add comments to each tests to explain what each test is for? I think that would be good.